### PR TITLE
Reduce too high encumbrance on clothing with pockets covering multiple body parts

### PR DESCRIPTION
**Pull request by Salty-Panda**
#### Summary

SUMMARY: Balance "Reduce too high encumbrance on clothing with pockets covering multiple body parts"

#### Purpose of change

I found 86 items of clothing that cover multiple body parts, have pockets, and have a "generic" encumbrance set for all body parts at once.
Example: SWAT armor has encumbrance 12-25 for torso, legs, AND arms, which scales with its 5L worth of pockets.

#### Describe the solution

I want to spread the difference between min and max encumbrance around all body parts with pockets 
(Example : SWAT armor torso 12-20, legs 12-17, arms 12-12) and in some cases lower max encumbrance further.
The basic ratio of 250ml per 1 encumbrance seems too low for some items.
I slightly changed this in a case of certain items, based on rather fluid rules, as described in #45621

#### Describe alternatives you've considered

None

#### Testing

Checked a few items in-game, mostly these with values changed.

#### Additional content
[list_of_items_to_change.txt](https://github.com/CleverRaven/Cataclysm-DDA/files/5625565/list_of_items_to_change.txt)

[changed items, changed values.txt](https://github.com/CleverRaven/Cataclysm-DDA/files/5648571/changed.items.changed.values.txt)

### CODE REVIEW

File: data/json/items/armor/coats.json [hunk 1]
```diff
@@ -15,10 +15,10 @@
     "symbol": "[",
     "looks_like": "coat_winter",
     "color": "yellow",
-    "covers": [ "torso", "arm_l", "arm_r" ],
-    "coverage": 100,
-    "encumbrance": 22,
-    "max_encumbrance": 30,
+    "armor_portion_data": [
+      { "covers": [ "torso" ], "coverage": 100, "encumbrance": [ 22, 26 ] },
```
Comment (anothersimulacrum) [inline data/json/items/armor/coats.json:19]: Please mention all cases like this where you change the max encumbrance to a different value than previous.

Comment (Mrzjadacz) [reply data/json/items/armor/coats.json:19]: In what way? A separate list in comment?

Comment (anothersimulacrum) [reply data/json/items/armor/coats.json:19]: Or in the opening post, yes.

**Comment by SirPendrak** [comment]
So far most changes seem ballanced, and my only nitpick would be swat armor. It got a buff to its armor not so long ago, and with nested containers changes they allowed you to put everything in backpack and run arround with that very good armor values for 12 encumbrance. Now it only got buffed. But different enc. distribution across different bodyparts is good. 

And since survivor suits changes are probably comming up here soon, i believe in most cases ther base encumbrance needs to be much higher, closer to their pre-nested containeers changes. For example, fitted heavy survivor suit now has 8-25 enc. Running it at 8 enc is crazy for that ammount of protection. 

Oh, and yet another nitpick. Keep in mind that some armors use varsize flag. I believe it either halves encumbrance value, or lowers it by 10, depending on which value is lower. Then for example leather duster is extreamly good protection for an ordinary clothing that will have 6 armor for 4 enc.

**Comment by mrkybe** [comment]
Hey since you're taking a look at this stuff, can you check the Plastic Canteen? it seems to be encumbering both legs...

**Comment by Mrzjadacz** [comment]
> Hey since you're taking a look at this stuff, can you check the Plastic Canteen? it seems to be encumbering both legs...

You're right, fixed.

**Comment by Mrzjadacz** [comment]
As far as I can tell, the failed tests aren't caused by my commit.
```../tests/field_test.cpp:44: FAILED:
  CHECK( count_fields( field_type ) == Approx( 8712 ).margin( 200 ) )
with expansion:
  8493 (0x212d) == Approx( 8712.0 )```
```../tests/weary_test.cpp:121: FAILED:
  CHECK( info.transition_minutes( 4, 3, 550_minutes ) == Approx( 550 ).margin( 0 ) )
with expansion:
Error:   520 (0x208) == Approx( 550.0 )```
```Build execution time has reached the maximum allowed time for your plan (60 minutes).```

**Pull request merged**